test/util/genesis/genesis.go

```diff
@@ -122,6 +122,9 @@ func (g *Genesis) WithKeyringAccounts(accs ...KeyringAccount) *Genesis {
 
 // AddAccount adds an existing account to the genesis.
 func (g *Genesis) AddAccount(account Account) error {
+	if err := account.ValidateBasic(); err != nil {
```

Comment:
are there any scenarios where this could consensus breaking? if so, are those scenarios relevant and do we need to document this somewhere?

Reply:
Why would this be consensus breaking? This is just about creating the genesis - I don't quite see how it affects actual running of a chain